modules/Bio/EnsEMBL/Analysis/Hive/Config/Projection_subpipeline_conf.pm

```diff
@@ -0,0 +1,518 @@
+########################
+# Misc setup info
+########################
+    'dbowner'                   => '' || $ENV{EHIVE_USER} || $ENV{USER},
+    'pipeline_name'             => '' || $self->o('production_name').'_'.$self->o('ensembl_release'),
+    'production_name'           => '', # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    'release_number'            => '' || $self->o('ensembl_release'),
+    'user_r'                    => '', # read only db user
+    'user'                      => '', # write db user
+    'password'                  => '', # password for write db user
+    'pipe_db_server'            => '', # host for pipe db
+    'dna_db_server'             => '', # host for dna db
+    'databases_server'          => '', # host for general output dbs
+    'pipe_db_port'              => '', # port for pipeline host
+    'dna_db_port'               => '', # port for dna db host
+    'databases_port'            => '', # port for general output db host
+    'uniprot_set'               => '', # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'output_path'               => '', # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'use_genome_flatfile'       => '1',# This will read sequence where possible from a dumped flatfile instead of the core db
+    'skip_projection'           => '0', # Will skip projection process if 1
+
+########################
+# Pipe and ref db info
+########################
+
+    'projection_source_db_name'         => '', # This is generally a pre-existing db, like the current human/mouse core for example
+    'projection_source_db_server'       => 'mysql-ens-mirror-1',
+    'projection_source_db_port'         => '4240',
+    'projection_source_production_name' => '',
+
+    'compara_db_name'   => 'leanne_ensembl_compara_95',
+    'compara_db_server' => 'mysql-ens-genebuild-prod-5',
+    'compara_db_port'   => 4531,
+
+    # The following might not be known in advance, since the come from other pipelines
+    # These values can be replaced in the analysis_base table if they're not known yet
+    # If they are not needed (i.e. no projection or rnaseq) then leave them as is
+    'pipe_db_name'  => $self->o('dbowner').'_'.$self->o('production_name').'_pipe_'.$self->o('release_number'),
+
+    'projection_lastz_db_name'     => $self->o('pipe_db_name'),
+    'projection_lastz_db_server'   => $self->o('pipe_db_server'),
+    'projection_lastz_db_port'     => $self->o('pipe_db_port'),
+
+    'dna_db_name'   => $self->o('dbowner').'_'.$self->o('production_name').'_core_'.$self->o('release_number'),
+
+    'projection_db_server'  => $self->o('databases_server'),
+    'projection_db_port'    => $self->o('databases_port'),
+
+    databases_to_delete => ['projection_db',],
```

Comment:
Could you please add the other databases that are created in this pipeline?

Reply:
It's hard to tell which are, this is the only one directly created using the normal method. It looks like the selected_projection_db is created within another analysis, should really be separated for clarity in future. I've added that one. The other DBs seem to be created elsewhere, though I'm not completely certain.